src/views/Home.vue

```diff
@@ -1,18 +1,58 @@
 <template>
-  <div class="home">
-<!--    <img alt="Vue logo" src="../assets/logo.png">-->
-    <HelloWorld msg="Welcome to Volo"/>
-  </div>
+    <div>
+        <NavigationBar/>
+        <div>
+            <h1>Popular Movies</h1>
+            <MediaHorizontalList class="MediaHorizontalList" :media-objs="popularMovies"/>
+        </div>
+        <div>
+            <h1>Trending Movies</h1>
+            <MediaHorizontalList class="MediaHorizontalList" :media-objs="trendingMovies"/>
+        </div>
+        <div>
+            <h1>Anticipated Movies</h1>
+            <MediaHorizontalList class="MediaHorizontalList" :media-objs="anticipatedMovies"/>
+        </div>
+        <div>
+            <h1>Box Office Hits</h1>
+            <MediaHorizontalList class="MediaHorizontalList" :media-objs="boxOfficeMovies"/>
+        </div>
+    </div>
 </template>
 
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator';
-import HelloWorld from '@/components/HelloWorld.vue'; // @ is an alias to /src
+    import {Component, Vue} from "vue-property-decorator";
+    import NavigationBar from "@/components/NavigationBar.vue";
+    import MediaHorizontalList from "@/components/MediaHorizontalList.vue";
+    import TraktApi from "@/services/TraktApi.ts";
+    import Movie from "@/models/Movie";
 
-@Component({
-  components: {
-    HelloWorld,
-  },
-})
-export default class Home extends Vue {}
+    @Component({
+        components: {
+            NavigationBar,
+            MediaHorizontalList,
+        },
+    })
+    export default class Home extends Vue {
+        private popularMovies: Movie[] = [];
+        private trendingMovies: Movie[] = [];
+        private anticipatedMovies: Movie[] = [];
+        private boxOfficeMovies: Movie[] = [];
+
+        private async mounted() {
+            const trakt = new TraktApi(); // todo
```

Comment:
Deliver a TraktApi to this component instead via dependency injection. This will allow for mocking and achieve inversion of control

Reply:
> Deliver a TraktApi to this component instead via dependency injection. This will allow for mocking and achieve inversion of control

[InveresifyJS](https://github.com/inversify/InversifyJS) looks like a good option.

Reply:
Will need to update [Decision Decisions](https://github.com/CoderKiwi/Volo/wiki/Design-decisions)

Reply:
This was worked on as part of #5